src/shared/styles/variables/palette.ts

```diff
@@ -12,6 +12,7 @@ export const palette = {
 
   on_surface: '#1a1c1e',
   on_surface_variant: '#42474e',
+  on_surface_variant_alpha8: 'rgba(66, 71, 78, 0.08)',
```

Comment:
I understand why you spelt it this way, so it's cool with me if we leave this in. Still, there is something to be said for consistency unless we want to do a refactor

```suggestion
  on_surface_variant_alfa8: 'rgba(66, 71, 78, 0.08)',
```

Reply:
Ah right… probably better to use the incorrect spelling for consistency 😑

Reply:
Oh… and in making this change, I noticed that the key already exists! 🙈 Good callout @sultanofcardio